README.md

```diff
@@ -23,24 +23,25 @@ Please note: We take Terraform's security and our users' trust very seriously. I
 ## Quick Starts
 
 - [Using the provider](https://registry.terraform.io/providers/hashicorp/aws/latest/docs)
-- [Provider development](docs/contributing)
+- [Provider development](https://hashicorp.github.io/terraform-provider-aws/)
 
 ## Documentation
 
-Full, comprehensive documentation is available on the Terraform website:
+Full, comprehensive documentation is available on the Terraform Registry:
+
+https://registry.terraform.io/providers/hashicorp/aws/latest/docs
 
-https://terraform.io/docs/providers/aws/index.html
 
 ## Roadmap
 
-Our roadmap for expanding support in Terraform for AWS resources can be found in our [Roadmap](ROADMAP.md) which is published quarterly.
+Our roadmap for expanding support in Terraform for AWS resources can be found in our [Roadmap](ROADMAP.md) which is published quarterly. 
 
 ## Frequently Asked Questions
 
-Responses to our most frequently asked questions can be found in our [FAQ](docs/contributing/faq.md )
+Responses to our most frequently asked questions can be found in our [FAQ](https://hashicorp.github.io/terraform-provider-aws/faq/)
 
 ## Contributing
 
 The Terraform AWS Provider is the work of thousands of contributors. We appreciate your help!
 
-To contribute, please read the contribution guidelines: [Contributing to Terraform - AWS Provider](docs/contributing)
+To contribute, please read the contribution guidelines: [Contributing to Terraform - AWS Provider](https://hashicorp.github.io/terraform-provider-aws/)
```

Comment:
This will actually take you to the landing page of the contributor guide, the fork contents are hosted here: https://breathingdust.github.io/terraform-provider-aws/

This currently 404's until this PR get's merged.